askama_derive/src/parser.rs

```diff
@@ -668,7 +670,29 @@ macro_rules! expr_prec_layer {
         }
     }
 }
-
+#[cfg(feature = "localization")]
+fn localize(i: &str) -> IResult<&str, Expr<'_>> {
+    let (tail, (_, _, message, attribute, args, _)) = tuple((
+        tag("localize"),
+        ws(tag("(")),
```

Comment:
Please add a cut here, e.g. wrap all following parsing instructions in a `cut(tuple((…)))`. This way the user gets a better error message if the content could not be parsed.

Reply:
Done